run/jobs/main_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestSuccessfulJob(t *testing.T) {
```

Comment:
Should there be TestJobSuccess with two cases: around FAIL_RATE empty/0 and FAIL_RATE 1? We're not testing the fault injection is honored.

Reply:
If FAIL_RATE is 1, it won't succeed.  Should I test that?  I test the actual failure function below with fail_rate of 1.

Reply:
I think testing it in the more complete way is nice but not required.